packages/paste-core/components/typography/__tests__/typography.test.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
```

Comment:
We should rename this file to `paragraph.test.tsx` and have different test files for each component for cleanliness. 👍